README.md

```diff
@@ -133,10 +132,15 @@ func send(report: DiagnosticsReport) {
 ```
 
 ### Using a custom UserDefaults type
-Simply set your user defaults instance by making use of:
+In order to use `UserDefaultsReporter`, you need to set instantiate it with the desired `UserDefaults` instance together with all the keys you would like to read, and use it in `DiagnosticsReporter.create(filename:using:filters:smartInsightsProvider)` to create a `DiagnosticsReport`.
```

Comment:
Nit: Should we update the title, for example, to: Using a user defaults reporter

Reply:
Nice catch - thanks! Addressed in https://github.com/WeTransfer/Diagnostics/pull/172/commits/d7eb8c6c4eefb77234115b3b15abf7817b67d137.